conf/script/build-ha-io

```diff
@@ -535,19 +535,7 @@ consul_rsc_add() {
 motr_kernel_rsc_add() {
     log "${FUNCNAME[0]}: Adding Motr kernel module to Pacemaker..."
     sudo pcs -f $cib_file resource create motr-kernel systemd:motr-kernel clone
-    sudo pcs -f $cib_file constraint order lnet-c1 then motr-kernel-clone
-    sudo pcs -f $cib_file constraint order lnet-c2 then motr-kernel-clone
-
-    # Make sure motr-kernel is always stopped before lnet-c1/2
-    # is started. Otherwise, some transition abort in between may
-    # prevent motr-kernel restart. This is a workaround suggested
-    # at https://bugs.clusterlabs.org/show_bug.cgi?id=5428#c5.
-    sudo pcs -f $cib_file constraint order stop motr-kernel-clone then \
-                                           start lnet-c1 \
-                                           kind=Optional symmetrical=false
-    sudo pcs -f $cib_file constraint order stop motr-kernel-clone then \
-                                           start lnet-c2 \
-                                           kind=Optional symmetrical=false
+    sudo pcs -f $cib_file constraint order lnet-clone then motr-kernel-clone
```

Comment:
`lnet-clone` is used here instead of lnet-c1. I would expect also changes where `lnet-clone` resource is created (or changed to clone), but don't see it.